test/evergreen.yml

```diff
@@ -4649,6 +4677,166 @@ tasks:
         vars:
           test_path: bench/wt2853_perf/wt2853_perf
 
+###################################################
+# Aggregate statistics for all performance tests  #
+###################################################
+
+  - name: aggregate-perf-stats
+    tags: ["aggregate-perf"]
```

Comment:
do we need this tag now?

Reply:
Nope, fixed that up in: [8c3543f](https://github.com/wiredtiger/wiredtiger/pull/8495/commits/8c3543fced10d30ac2d76639f98744a6e826e4b8)